server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -365,13 +428,15 @@ public void addInvalidAgeUser() throws IOException {
         + "}";
     mockReq.setBodyContent(testNewUser);
     mockReq.setMethod("POST");
-    Context ctx = ContextUtil.init(mockReq, mockRes, "api/users");
 
-    assertThrows(BadRequestResponse.class, () -> {
+    Context ctx = mockContext("api/users");
+
+    assertThrows(ValidationException.class, () -> {
       userController.addNewUser(ctx);
     });
   }
 
+  @Test
```

Comment:
Lol :) nice catch!

Reply:
Yeah, there were _two_ of these "test" methods that didn't have the `@Test` annotation. Oops.